docker-compose.yml

```diff
@@ -11,8 +11,6 @@ services:
   jenkins_build:
     image: edxops/jenkins_build:latest
     container_name: jenkins_build
-    volumes:
-        - jenkins_build:/var/lib/jenkins
```

Comment:
Yes. I didn't know that persisting it was very desirable, as the DSL code is all in Github. Perhaps I am not in sync with the way that most developers use this container.

Reply:
I'm not sure either how most developers use this.  I just had used it to local testing, so I didn't need this volume, but added it in originally because there was an early notion that other openedx administrators could just launch this container to get a quick and dirty user retirement implementation.  It is becoming clear that those administrators may not be using this container, and we pivoted to not officially supporting using this container for that use case, so I'm okay with whatever outcome you decide.